custom_integrations/creasoldombus/README.md

```diff
@@ -0,0 +1,12 @@
+# Creasol Dombus icons and logo for Home Assistant
```

Comment:
This file does not belong here, please remove this file.

Reply:
Ok, modifications have been done.
Thanks a lot for the icon quality suggestion.
Paolo